configs/sst_rhel_lightspeed.yaml

```
@@ -0,0 +1,24 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: RHEL Lightspeed
+  description: Command Line Assistant powered by RHEL Lightspeed.
+  maintainer: sst_rhel_lightspeed
+  packages:
+    - command-line-asssitant
+  package_placeholders:
+    - srpm_name: command-line-assistant
+      rpms:
+        - rpm_name: command-line-assistant
+          description: Command Line Assistant
+          dependencies:
+            - tomli
```

Based on your copr build, and that only RHEL 10+ is tracked in Content Resolver:
```suggestion
            - python3
```